src/AbstractServer.php

```diff
@@ -72,20 +69,31 @@ protected function _buildCallback(Reflection\AbstractFunction $reflection)
         return $callback;
     }
 
+    /**
+     * Build callback for method signature
+     *
+     * @deprecated Since 2.7.0; method will be removed in 3.0, use
+     *             buildCallback() instead.
+     * @param  Reflection\AbstractFunction $reflection
+     * @return Method\Callback
+     */
+    // @codingStandardsIgnoreStart
+    protected function _buildCallback(Reflection\AbstractFunction $reflection)
+    {
+    // @codingStandardsIgnoreEnd
+        return $this->buildCallback($reflection);
+    }
+
     /**
      * Build a method signature
      *
-     * @deprecated Since 2.7.0; method will be renamed to remove underscore
-     *     prefix in 3.0.
      * @param  Reflection\AbstractFunction $reflection
      * @param  null|string|object $class
      * @return Method\Definition
      * @throws Exception\RuntimeException on duplicate entry
      */
-    // @codingStandardsIgnoreStart
-    protected function _buildSignature(Reflection\AbstractFunction $reflection, $class = null)
+    final protected function buildSignature(Reflection\AbstractFunction $reflection, $class = null)
```

Comment:
Not sure what has happened here - but I cannot see old `_buildSignature` as a proxy to the new one.

BTW - why `final` here?

Reply:
@arueckauer — we need the `_buildSignature()` proxy method as well, as extensions consume it.

@michalbundyra `final` is good here, as what it returns has to be done in a very specific way, and should not be overridden in extensions. If they want to do something different, they should provide their own method.

Reply:
Restored `_buildSignature()` as proxy method.